service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/AdditionalApplicationsUploadedEmailContentProviderTest.java

```diff
@@ -0,0 +1,126 @@
+import uk.gov.hmcts.reform.fpl.model.common.AdditionalApplicationsBundle;
+import uk.gov.hmcts.reform.fpl.model.common.C2DocumentBundle;
+import uk.gov.hmcts.reform.fpl.model.common.OtherApplicationsBundle;
+import uk.gov.hmcts.reform.fpl.model.notify.BaseCaseNotifyData;
+import uk.gov.hmcts.reform.fpl.model.notify.additionalapplicationsuploaded.AdditionalApplicationsUploadedTemplate;
+import uk.gov.hmcts.reform.fpl.utils.FixedTimeConfiguration;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Locale;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.enums.SupplementType.C13A_SPECIAL_GUARDIANSHIP;
+import static uk.gov.hmcts.reform.fpl.enums.SupplementType.C20_SECURE_ACCOMMODATION;
+import static uk.gov.hmcts.reform.fpl.enums.TabUrlAnchor.OTHER_APPLICATIONS;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
+
+@ContextConfiguration(classes = {AdditionalApplicationsUploadedEmailContentProvider.class,
+    FixedTimeConfiguration.class})
+class AdditionalApplicationsUploadedEmailContentProviderTest extends AbstractEmailContentProviderTest {
+
+    @Autowired
+    private AdditionalApplicationsUploadedEmailContentProvider additionalApplicationsUploadedEmailContentProvider;
+
+    private static final LocalDateTime HEARING_DATE = LocalDateTime.now().plusMonths(3);
+
+    private static final String HEARING_CALLOUT = "hearing " + HEARING_DATE
+        .toLocalDate().format(DateTimeFormatter.ofLocalizedDate(FormatStyle.MEDIUM).localizedBy(Locale.UK));
+
+    @Test
+    void shouldReturnExpectedMapWithGivenCaseDetails() {
+        CaseData caseData = buildCaseData();
+
+        AdditionalApplicationsUploadedTemplate expectedParameters =
+            getAdditionalApplicationsUploadedTemplateParameters();
+        AdditionalApplicationsUploadedTemplate actualParameters = additionalApplicationsUploadedEmailContentProvider
+            .getNotifyData(caseData);
+
+        assertThat(actualParameters).usingRecursiveComparison().isEqualTo(expectedParameters);
+    }
+
+    @Test
+    void shouldReturnExpectedPbaPaymentNotTakenNotification() {
+        CaseData caseData = buildCaseData();
+
+        BaseCaseNotifyData expectedParameters = BaseCaseNotifyData.builder()
+            .caseUrl(caseUrl(CASE_REFERENCE, OTHER_APPLICATIONS))
+            .build();
+
+        BaseCaseNotifyData actualParameters = additionalApplicationsUploadedEmailContentProvider
+            .getPbaPaymentNotTakenNotifyData(caseData);
+
+        assertThat(actualParameters).isEqualTo(expectedParameters);
+    }
+
+    private AdditionalApplicationsUploadedTemplate getAdditionalApplicationsUploadedTemplateParameters() {
+        return AdditionalApplicationsUploadedTemplate.builder()
+            .callout("Smith, 12345, " + HEARING_CALLOUT)
```

Comment:
Test is failing because callout here needs changed to ^Smith, 12345, " + HEARING_CALLOUT

Reply:
Made this change in anticipation of [PR 2375](https://github.com/hmcts/fpl-ccd-configuration/pull/2375) which is meant to fix a double callout issue. Planning to switch this branch to merge into Master so I can get that fix

Reply:
oh yes of course, makes sense :)